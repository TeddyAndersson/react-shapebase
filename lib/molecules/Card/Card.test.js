import React from 'react';
import Card from './Card'
import renderer from 'react-test-renderer';

const lorem = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."

test('should render Card.Header', () => {
  const component = renderer.create(
    <Card.Header>
      {lorem}
    </Card.Header>
  );
  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
})

test('should render Card.Content', () => {
  const component = renderer.create(
    <Card.Content>
      {lorem}
    </Card.Content>
  );
  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
})

test('should render Card.Footer', () => {
  const component = renderer.create(
    <Card.Footer>
      {lorem}
    </Card.Footer>
  );
  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
})

test('should render Card', () => {
  const component = renderer.create(
    <Card>
      <Card.Header>
        {lorem}
      </Card.Header>
      <Card.Content>
        {lorem}
      </Card.Content>
      <Card.Footer>
        {lorem}
      </Card.Footer>
    </Card>
  );
  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
})
