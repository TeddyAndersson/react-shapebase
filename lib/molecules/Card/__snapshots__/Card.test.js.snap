// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Card 1`] = `
<div
  className="sc-EHOje dSWRWp sc-bwzfXH lipQua sc-bdVaJa fSzclR"
>
  <header
    className="sc-htpNat jdjlcw"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
  </header>
  <section
    className="sc-bxivhb hLXbmz"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
  </section>
  <footer
    className="sc-ifAKCX hcUVps"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
  </footer>
</div>
`;

exports[`should render Card.Content 1`] = `
<section
  className="sc-bxivhb hLXbmz"
>
  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
</section>
`;

exports[`should render Card.Footer 1`] = `
<footer
  className="sc-ifAKCX hcUVps"
>
  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
</footer>
`;

exports[`should render Card.Header 1`] = `
<header
  className="sc-htpNat jdjlcw"
>
  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
</header>
`;
